Application/src/visuals/handlers/Animation.java

```diff
@@ -0,0 +1,39 @@
+package visuals.handlers;
+
+import visuals.DiagramView;
+import visuals.Draw;
+
+/**
+ * Handles refreshing and animating draw objects.7
+ * @version 0.1
+ * @author Pontus Laestadius
+ */
+public class Animation extends Thread {
+    Draw item;
+
+    public Animation(Draw item) {
+        this.item = item;
+    }
+
+    @Override
+    public void run() {
+         loop();
+    }
+
+    private void loop() {
```

Comment:
What does this do?

Reply:
It loops. Pretty self explanatory with the document header.